### ISSUE TITLE
Mention XLSX.jl in Alternatives section in readme?

### ISSUE THREAD

**Issue by cmcaine** [queryverse/ExcelReaders.jl#69]
XLSX.jl is a pure-julia implementation for parsing xlsx files. In my
testing it doesn't work for some sheets that are produced by strange and
bad software, but should work for most use cases.

**Comment by davidanthoff** [comment]
Yes, definitely! My hope is actually that one day I can switch out the backend for ExcelFiles.jl to use XLSX.jl and https://github.com/queryverse/LibXLS.jl. But I think both of those would have to be more robust first... But yes, mentioning it would make sense. PR would be most welcome :)

**Comment by davidanthoff** [comment]
The new version on `main` uses XLSX and LibXLS.
